### ISSUE TITLE
Removing the main folder breaks the extension

### ISSUE THREAD

**Issue by cengels** [perfect-things/perfect-home#85]
The following screen appears when the user manually deletes the folder marked as the extension's main folder:

![image](https://user-images.githubusercontent.com/31248098/79111930-c48a3f80-7d7d-11ea-973b-0b985065695a.png)

In this state, neither the settings nor the search can be opened. As such, the main folder cannot be changed and the extension is in a permanently broken state. The only way to resolve this is to reinstall the extension, thus clearing all its data.

The following errors pop up in the console when you try to open the settings in this state:

```
TypeError: can't access property "length", t[2] is undefined in index.js:1:23234
Error: "can't access property "guid", node is null" index.js:1:19394
    Nt moz-extension://18fbc609-cc2a-47cb-9f32-95dfbdb5dd27/index.js:1
    (Async: promise callback)
    Nt moz-extension://18fbc609-cc2a-47cb-9f32-95dfbdb5dd27/index.js:1
    g moz-extension://18fbc609-cc2a-47cb-9f32-95dfbdb5dd27/index.js:7
    d moz-extension://18fbc609-cc2a-47cb-9f32-95dfbdb5dd27/index.js:7
    subscribe moz-extension://18fbc609-cc2a-47cb-9f32-95dfbdb5dd27/index.js:1
    m moz-extension://18fbc609-cc2a-47cb-9f32-95dfbdb5dd27/index.js:7
    fire moz-extension://18fbc609-cc2a-47cb-9f32-95dfbdb5dd27/index.js:1
    fire moz-extension://18fbc609-cc2a-47cb-9f32-95dfbdb5dd27/index.js:1
    <anonymous> moz-extension://18fbc609-cc2a-47cb-9f32-95dfbdb5dd27/index.js:1
```

**Comment by cengels** [comment]
This appears to be the same issue as the one mentioned in #82 and #83.

**Comment by tborychowski** [comment]
I've managed to reproduce this one and I've got the fix now.

**Comment by tborychowski** [comment]
This should be fixed in the latest version 0.10.0.
Please reopen the issue if you still see the problem.
